symbols/arm9.yml

```diff
@@ -994,7 +994,19 @@ arm9:
         r0: [output] pointer to an IO struct {ptr, len}
         r1: file path string pointer
         r2: flags
-    - name: GetDebugFlag
+    - name: InitDebug
+      address:
+        NA: 0x200C0D4
+      description: |-
+        Would have initialized debugging-related things, if they were not removed.
+        As for the release version, does nothing but set DEBUG_IS_INITIALIZED to true.
```

Comment:
Is there supposed to be a global with this name?

Reply:
The sync script doesn't support globals yet. Adding it manually.